### PR TITLE
maven source plugin

### PR DESCRIPTION
**Pull request by marmoure**
```xml
<plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-source-plugin</artifactId>
    <version>3.2.0</version>
    <executions>
        <execution>
            <id>attach-sources</id>
            <phase>verify</phase>
            <goals>
                <goal>jar-no-fork</goal>
            </goals>
        </execution>
    </executions>
</plugin>
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -92,6 +93,20 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <version>3.2.0</version>
```
Comment (adamretter) [inline pom.xml:99]: Please use the latest version

**Review by adamretter — CHANGES_REQUESTED** [review]
Needs manifest entries. Also should use the latest version

File: pom.xml [hunk 2]
```diff
@@ -92,6 +93,20 @@
...
+                <executions>
+                    <execution>
+                        <id>attach-sources</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>jar-no-fork</goal>
```
[the remark this answers is not in the document]
Comment (marmoure) [reply pom.xml:105]: i could not fine documentation for that

[the remark this answers is not in the document]
Comment (adamretter) [reply pom.xml:105]: @marmoure there is an example here for you - https://github.com/digital-preservation/utf8-validator/blob/master/pom.xml#L226

**Pull request merged**